posthog/api/hog_function.py

```diff
@@ -91,3 +100,43 @@ class HogFunctionViewSet(TeamAndOrgViewSetMixin, LogEntryMixin, ForbidDestroyMod
 
     def get_serializer_class(self) -> type[BaseSerializer]:
         return HogFunctionMinimalSerializer if self.action == "list" else HogFunctionSerializer
+
+    @action(detail=False, methods=["GET"])
+    def icons(self, request: Request, *args, **kwargs):
+        query = request.GET.get("query")
+        if not query:
+            return Response([])
+
+        if not settings.LOGO_DEV_TOKEN:
+            raise serializers.ValidationError("LOGO_DEV_TOKEN is not set")
+
+        res = requests.get(f"https://search.logo.dev/api/icons?token={settings.LOGO_DEV_TOKEN}&query={query}")
```

Comment:
do we want any sanitization here? otherwise a search with `tom & jerry` will cause problems